x/operator/keeper/abci.go

```diff
@@ -90,10 +107,90 @@ func (k *Keeper) UpdateVotingPower(ctx sdk.Context, avsAddr string) error {
 	if err != nil {
 		return err
 	}
+
+	// TODO: Consider not addressing the dogfood AVS, as its historical voting power
+	// has already been stored by CometBFT.
+
+	// set voting power snapshot
+	// When the snapshot helper does not exist, it represents the initial state of AVS,
+	// where no snapshot information has been stored. Therefore, it is necessary to store
+	// both the snapshot and the helper information.
+	var snapshotHelper operatortypes.SnapshotHelper
```

Comment:
```suggestion
	snapshotHelper := operatortypes.SnapshotHelper{}
```

Reply:
Thanks, it will be updated in the next commit.